CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 ## UNRELEASED
 
 <!-- Add changelog entries for new changes under this section -->
+<!--
+* **Improvement**
+   * Add support JS modules ([#252](https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/252), [@jlopezxs](https://github.com/jlopezxs)))
+-->
```

Comment:
Ah, you don't need to add the new changelog entry in a comment :relaxed:. Only the "Add changelog entries" line needs to be a comment, new entries can be listed as is.

Also, what do you think if the wording was changed to "Add support for .mjs output files"? I fear that "support for JS modules" would cause people to think that `webpack-bundle-analyzer` didn't support JS modules as sources before. Because this PR is about supporting `.mjs` output files, right?